ktbridge/plugin-gradle/src/main/kotlin/app/cash/quickjs/ktbridge/plugin/KtBridgeGradleExtension.kt

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (C) 2020 Brian Norman
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package app.cash.quickjs.ktbridge.plugin
+
+import org.gradle.api.file.RegularFileProperty
+import org.gradle.api.model.ObjectFactory
+import org.gradle.api.provider.Property
+
+open class KtBridgeGradleExtension(objects: ObjectFactory) {
```

Comment:
I don't see any usage of the extension. Delete it all until needed?

Reply:
Deleted!